guide/designing-products/common-user-flows.md

```diff
@@ -251,13 +251,19 @@ See the [onboarding section](/guide/onboarding/introduction/) to learn more on t
    layout = "float-left"
 %}
 
-It is essential to distinguish between wallet software and the wallet representing the user's "account." This particular experience may be almost invisible or highly elaborate. While some mobile wallet applications choose to create and secure a wallet during the first launch automatically, other wallets provide extensive configuration wizards.
+By _wallet_, we normally mean the wallet [_application_](/guide/glossary/wallet/#wallet-application). Here we will talk about creating the [_Bitcoin wallet_](/guide/glossary/wallet/#bitcoin-wallet), the private key(s) that the software interacts with to control funds on the Bitcoin blockchain.
```

Comment:
This is quite confusing to read, are we creating the wallet software? It may be too far gone for this but I like to think:

Bitcoin Wallet = The application as a whole that constructs / edits / broadcasts transactions.
Bitcoin Account = Private keys generated and used for a specific purpose.

Reply:
Agree it's not entirely satisfying, but it's how we have structured the glossary and how this page referred to it in the past. I suggest we leave it like this for the Lightning changes, and if we want to change to use account (which has other drawbacks imo.) we do that change across all relevant pages.

Reply:
```suggestion
By _wallet_, we normally mean the [_wallet application_](/guide/glossary/wallet/#wallet-application). In this context, we will instead talk about creating the [_bitcoin wallet_](/guide/glossary/wallet/#bitcoin-wallet) (the data structure which stores and manages the user's private key(s), which the software interacts with to control funds on the Bitcoin blockchain.
```

Reply:
I agree with @danielnordh we should keep the terminology consistent. I also agree with @Bosch-0 it's a little confusing to read.

I suggested some sentence structure changes. I think including "in this context" and "instead" will help clarify that we are talking about something different. 
  
Additionally, I borrowed the description "data structure which stores and manages the user's private key(s)" from [chapter 5 of bitcoinbook](https://github.com/bitcoinbook/bitcoinbook/blob/develop/ch05.asciidoc). But that might conflict with the definition in the glossary.
  
Also, I noticed that the main header nav obscures "bitcoin wallet" and "wallet application" when clicking on the glossary links, which may confuse people if they click on those links. It doesn't happen if on the main glossary page, only on the "Wallets" glossary page. I'll open a separate issue for that.

Reply:
Opened #500 for the issue I describe above with the term being obscured under header